src/resolvers/resourceResolvers.ts

```diff
@@ -130,4 +117,25 @@ export const resolvers = {
       });
     },
   },
+  TaxonomyContext: {
+    async crumbs(taxonomyContext: TaxonomyContext, _: any, context: ContextWithLoaders): Promise<GQLTaxonomyCrumb[]> {
+      const parentNodes = await context.loaders.nodeLoader.loadMany(
+        taxonomyContext.parentContextIds.map((contextId) => ({ contextId })),
+      );
+      const crumbs = parentNodes
+        .filter((node) => node.length > 0)
+        .map((node) => {
+          const parent = node[0]!;
+          const entity = nodeToTaxonomyEntity(parent, context.language);
+          return {
+            id: entity.id,
+            contextId: entity.contextId ?? "",
```

Comment:
Kun for å sikre mapping siden node.contextId er optional, men siden vi henter noder basert på contextId vil denne alltid returnere en verdi.